openjdk8/src/test/resources/structure.yaml

```diff
@@ -8,6 +8,11 @@ commandTests:
 - name: 'correct java version is installed'
   command: ['java', '-version']
   expectedError: ['openjdk version "1\.${openjdk.version.major}.*${openjdk.version.minor}"']
+- name: 'ensure java7 is not installed as a dependency accidentally'
+  setup: [[ 'chmod', '+x', '/workspace/openjdk8/src/test/workspace/test-no-java-7.bash' ]]
+  command: [ '/workspace/openjdk8/src/test/workspace/test-no-java-7.bash' ]
+  expectedError: ['WARNING.*apt does not have a stable CLI interface yet.']
+  expectedOutput: ['OK.* is not installed']
```

Comment:
Agreed, thank you, for catching this! 

I tried it out with java 8 and it would have not caught it. 
I committed a new version that explicitly states "NOT OK. xxxx is installed" and now the test matches that as the ``excludedOutput``. I also removed ``expectedError`` as it is not really expected, just meant to be an explanatory piece - however it might just be more distracting than anything.